TA.ArduinoPowerController.Server/ServerStatusDisplay.cs

```diff
@@ -1,6 +1,6 @@
 // This file is part of the TA.ArduinoPowerController project
 //
-// Copyright � 2016-2017 Tigra Astronomy, all rights reserved.
+// Copyright � 2016-2017 Tigra Astronomy, all rights reserved.
```

Comment:
The copyright symbol seems to have been corrupted. Make sure you are saving files as UTF-8 with BOM.